components/EntryUnit.vue

```diff
@@ -0,0 +1,24 @@
+<script setup lang="ts">
+defineProps({
+  content: String,
+  createdAt: String,
+});
```

Comment:
By the way you could also define props differently:

https://vuejs.org/api/sfc-script-setup.html#type-only-props-emit-declarations

![Screenshot 2024-03-04 at 10 49 16](https://github.com/boyeonihn/manyaday/assets/499898/3ff90d16-ee5e-4e38-beda-41513813079a)

You can get a better typing! (no need for ! after createdAt)

Reply:
Ah! I just read about this today on vue's official docs. 😁

Reply:
great! :D